projects/core/styles/theme/appearance/close.less

```diff
@@ -0,0 +1,14 @@
+@import '../../taiga-ui-local.less';
+
+[tuiAppearance][data-appearance='close'] {
```

Comment:
This looks a lot like `neutral` appearance, let's move it there and just override color